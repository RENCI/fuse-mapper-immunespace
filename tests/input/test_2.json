[
  {
    "id": "1",
    "resource": {
        "experimentData": {},
        "exprs": {},
        "featureNames": {},
        "pData": {},
        "phenoMetadata": {},
        "system": "HUGO"
    },
    "resourceType": "eset"
  },
  {
    "id": "2",
    "resource": {
        "experimentData": {},
        "exprs": {},
        "featureNames": {},
        "pData": {},
        "phenoMetadata": {},
        "system": "HUGO"
    },
    "resourceType": "eset"
  }
]
