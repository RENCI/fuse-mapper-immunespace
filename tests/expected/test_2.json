[
{
    "objectId": "1",
    "participants": [
	{
	    "participantId": "test_empty",
	    "values": [
		{
		    "title": "Study Time Collected",
		    "id": "study_time_collected",
		    "certitude": 2,
		    "group": "pData",
		    "how": "The value was provided by the digital object server.",
		    "timestamp": "2020-08-01T14:29:15.453Z",
		    "variableValue": { "value":0, "units": "Days" }
		},
		{ 
		    "title": "Gene expression",
		    "id": "FUSE:expression",
		    "certititude": 2,
		    "how": "From Immunescape microarrays",
		    "variableValue": {
			"value": {
			    "system": "entrez",
			    "aliquots": [ {
				"aliquotId":"test_empty_aliquot1",
				"value": {
				    "1": "1.0",
				    "503538": "2.0",
				    "29974": "3.0"
				}
			    },
			    {
				"aliquotId":"test_empty_aliquot2",
				"value": {
				    "1": "1.5",
				    "503538": "2.5",
				    "29974": "3.5"
				}
			    } ]
			}
		    }
		}
	    ]
	}
    ]
},
{
    "objectId": "cellfie_group2",
    "participants": [ 
	{
	    "participantId": "SUB112834.269",
	    "values": [
		{
		    "title": "Study Time Collected",
		    "id": "study_time_collected",
		    "certitude": 2,
		    "group": "pData",
		    "how": "The value was provided by the digital object server.",
		    "timestamp": "2020-08-01T14:29:15.453Z",
		    "variableValue": { "value":0, "units": "Days" }
		},
		{
		    "title": "Gene expression",
		    "id": "FUSE:expression",
		    "certititude": 2,
		    "how": "From Immunescape microarrays",
		    "variableValue": {
			"value": {
			    "system": "entrez",
			    "aliquots": [ {
				"aliquotId":"BS586128",
				"value": {
				    "1": "3.78615018912845",
				    "503538": "3.68295613722495",
				    "29974": "3.82804625098162"
				}
			    } ]
			}
		    }
		}
	    ]
	},
	{
	    "participantId": "SUB112834.269",
	    "values": [
		{
		    "title": "Study Time Collected",
		    "id": "study_time_collected",
		    "certitude": 2,
		    "group": "pData",
		    "how": "The value was provided by the digital object server.",
		    "timestamp": "2020-08-01T14:29:15.453Z",
		    "variableValue": { "value": 7, "units": "Days" }
		},
		{
		    "title": "Gene expression",
		    "id": "FUSE:expression",
		    "certititude": 2,
		    "how": "From Immunescape microarrays",
		    "variableValue": {
			"system": "entrez",
			"aliquots":[ {
			    "aliquotId":"BS586240",
			    "value": {
				"1": "3.15087086107396",
				"503538": "3.22078940893019",
				"29974": "3.06783937011117"
			    }
			} ]
		    }
		}
	    ]
	},
	{
	    "participantId": "SUB112832.269",
	    "values": [
		{
		    "title": "Study Time Collected",
		    "id": "study_time_collected",
		    "certitude": 2,
		    "group": "pData",
		    "how": "The value was provided by the digital object server.",
		    "timestamp": "2020-08-01T14:29:15.453Z",
		    "variableValue": { "value": 0, "units": "Days" }
		},
		{
		    "title": "Gene expression",
		    "id": "FUSE:expression",
		    "certititude": 2,
		    "how": "From Immunescape microarrays",
		    "variableValue": {
			"system": "entrez",
			"aliquots": [ {
			    "aliquotId":"BS586131",
			    "value": {
				"1": "3.68940264786164",
				"503538": "3.51391883064394",
				"29974": "3.50394031553657"
			    }
			} ]
		    }
		}
	    ]
	}
    ]
}
]
